scene/animation/animation_mixer.cpp

```diff
@@ -2452,6 +2607,11 @@ void AnimationMixer::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "reset_on_save", PROPERTY_HINT_NONE, ""), "set_reset_on_save_enabled", "is_reset_on_save_enabled");
 	ADD_PROPERTY(PropertyInfo(Variant::NODE_PATH, "root_node"), "set_root_node", "get_root_node");
 
+	ADD_GROUP("Fps Lod", "fps_lod");
```

Comment:
```suggestion
	ADD_GROUP("FPS LOD", "fps_lod");
```
Unless these are something else than "frames per second" and "level of detail"